### DISCUSSION TITLE
Best Integration for Invoicing

### DISCUSSION THREAD

**Discussion by mr2017x** [kimai/kimai#3385] [General]
Hi Guys,

i am trying to combine hourly based invoices with recurring standard items which i am in search of a matching software - maybe box billing (any hints for other appreciated) - how could integration be done? is there already some preparation for?

**Comment by kevinpapst** [comment]
What is you idea on this? Use another invoicing software and sync the timesheet records towards them?

**Comment by mr2017x** [comment]
Perhabs, i am on evaluation for better invoicing flows, but also into how to integrate it. Which also means just set up one customer once over all systems.
